Receipt/Sources/ListReceiptPresenter.swift

```diff
@@ -101,6 +179,10 @@ final class ListReceiptPresenter {
                 }
                 strongSelf.isLoadInProgress = false
                 view.hideLoading()
+                if strongSelf.isFirstTimeLoadingPage {
+                    view.reloadTableViewHeader()
+                    strongSelf.isFirstTimeLoadingPage = false
+                }
```

Comment:
move `view.reloadTableViewHeader()` to ` func listAllAvailableReceipts` that will be called only once during page load. Then you won't need this `var isFirstTimeLoadingPage`. Also you are creating header once you have all the information about prepaid cards so it makes sense to move this logic there